smoketest.md

```diff
@@ -0,0 +1,102 @@
+# Smoke Test
+
+For a new system setup, follow the installation steps in [install.md](install.md) 
+
+## Setup
+
+#### Start local Atlas Data Federation
+Set the following environment variables:
+```
+export ADF_TEST_LOCAL_USER=<adf username>
+export ADF_TEST_LOCAL_PWD=<adf password>
+export ADF_TEST_LOCAL_AUTH_DB=<local auth db>
+export ADF_TEST_LOCAL_HOST=<local host address>
+export MDB_TEST_LOCAL_PORT=<MongoDB port>
+```
```

Comment:
Do you think it's worth adding how to add these on Windows?

```suggestion
Set the following environment variables:

Windows:
```
```sh
setx ADF_TEST_LOCAL_USER "<adf username>"
setx ADF_TEST_LOCAL_PWD "adf password>"
setx ADF_TEST_LOCAL_AUTH_DB "<local auth db>"
setx ADF_TEST_LOCAL_HOST "<local host address>"
setx MDB_TEST_LOCAL_PORT "<MongoDB port>"
```

POSIX
```
export ADF_TEST_LOCAL_USER=<adf username>
export ADF_TEST_LOCAL_PWD=<adf password>
export ADF_TEST_LOCAL_AUTH_DB=<local auth db>
export ADF_TEST_LOCAL_HOST=<local host address>
export MDB_TEST_LOCAL_PORT=<MongoDB port>
```
```

Reply:
Okay, suggestions with internal code fences don't seem to play well together in the UI!

Reply:
Sounds good, added the section for Windows